ci/test_linux.sh

```diff
@@ -36,6 +36,7 @@ else
     TEST_IMAGE_NAME="${TEST_IMAGE_NAMES[$TARGET_DOCKER_TEST_IMAGE]}"
 fi
 echo $TEST_IMAGE_NAME
+echo "$NEXUS_PASSWORD" | docker login --username jenkins --password-stdin
```

Comment:
we have script to login to docker internally https://github.com/snowflakedb/libsnowflakeclient/blob/master/ci/scripts/login_internal_docker.sh
can we use it instead?

Reply:
yes, this line was not working, I will try with the script